Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[required]
+python_version = "3.7"
+
+[packages]
+django = ">=2.0"
+pymysql = ">=0.7.11"
+pytz = "==2017.3"
```

Comment:
I believe `pytz` can be removed from here since it is just a dependency of the django package anyway. It was in the `requirements.txt` file originally, but that was just because it was a pinned dependency, the `requirements-minimal.txt` did not have it.

Reply:
Removed.